types/src/content/index.ts

```diff
@@ -39,6 +39,7 @@ export class EnglishAuction extends JoyStructDecorated({
   top_bid: Option.with(EnglishAuctionBid),
   whitelist: BTreeSet.with(MemberId),
   extension_period: BlockNumber,
+  auction_duration: BlockNumber,
```

Comment:
We need to revisit this `EnglishAuction` type as it has been changed recently in the runtime. See https://github.com/Joystream/joystream/blob/rhodes/runtime-modules/content/src/nft/types.rs#L134 and update the type accordingly, please. Note that the order of parameters matters.